regional/onboarding/README.md

```diff
@@ -26,22 +26,20 @@ No modules.
 | [kubernetes_role_binding_v1.namespace_admin](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/role_binding_v1) | resource |
 | [kubernetes_role_v1.namespace_admin](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/role_v1) | resource |
 | [kubernetes_service_account_v1.workload_identity](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/service_account_v1) | resource |
-| [google_service_account.namespace_admin](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/service_account) | data source |
 | [google_service_account.workload_identity](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/service_account) | data source |
 
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| <a name="input_google_service_account"></a> [google\_service\_account](#input\_google\_service\_account) | The optional email address of the pre-existing Google service account to use for the namespace administrator | `string` | `""` | no |
-| <a name="input_namespace_admin"></a> [namespace\_admin](#input\_namespace\_admin) | The namespace administrator service account name, required if google\_service\_account is not set | `string` | `""` | no |
+| <a name="input_google_service_account"></a> [google\_service\_account](#input\_google\_service\_account) | The email address of the pre-existing Google service account to use for the namespace administrator | `string` | n/a | yes |
 | <a name="input_namespaces"></a> [namespaces](#input\_namespaces) | A map of namespaces | <pre>map(object({<br>    istio_injection = optional(string, "disabled")<br>  }))</pre> | n/a | yes |
 | <a name="input_project_id"></a> [project\_id](#input\_project\_id) | The ID of the project in which the resource belongs | `string` | n/a | yes |
+| <a name="input_workload_identity_service_account_emails"></a> [workload\_identity\_service\_account\_emails](#input\_workload\_identity\_service\_account\_emails) | A map of workload identity service account emails for each namespace | `map(string)` | n/a | yes |
```

Comment:
This is helpful but it would have been better to comment on the variable description. These docs are generated by terraform-docs automatically based on the variables, outputs, resources, modules and providers. I've updated the variable description.